tools/backups2datalad/util.py

```diff
@@ -281,7 +281,7 @@ def dataset_files(dspath: Path) -> Iterator[str]:
     )
     while files:
         p = files.popleft()
-        if p.is_file():
+        if p.is_file() or p.is_symlink():
```

Comment:
oh , I did feel that this absence of `os.path.lexists` would be our ghost in the land of datalad datasets, but was hoping not to see it this way ;) Thanks!  I will merge and test now on 000026 first